aggregator/src/app/EthereumService.ts

```diff
@@ -331,6 +363,21 @@ export default class EthereumService {
     throw new Error("Expected return or throw from attempt loop");
   }
 
+  async GasConfig() {
+    const block = await this.wallet.provider.getBlock("latest");
+    const previousBaseFee = block.baseFeePerGas;
+    assert(previousBaseFee !== null && previousBaseFee !== nil);
+
+    return {
+      maxFeePerGas: previousBaseFee
+        .add(
+          previousBaseFee.mul(env.PREVIOUS_BASE_FEE_PERCENT_INCREASE).div(100),
```

Comment:
Might be nice to give this a variable for readability. ex:

```
const feeIncrease = previousBaseFee.mul(env.PREVIOUS_BASE_FEE_PERCENT_INCREASE).div(100);
```

Reply:
Done https://github.com/web3well/bls-wallet/pull/438